Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
Can you configure Puma following Heroku recommendations? https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#config